napari/layers/points/points.py

```diff
@@ -696,23 +696,15 @@ def edge_color(self, edge_color):
 
     @property
     def edge_color_cycle(self):
-        """Union[list, np.ndarray, cycle] :  Color cycle for edge_color.
-        Can be a list of colors or a cycle of colors
+        """Union[list, np.ndarray] :  Color cycle for edge_color.
```

Comment:
maybe while we're at it we can turn these into proper return annotations?

```python
    def edge_color_cycle(self) -> Union[list, np.ndarray]:
        """Color cycle for edge_color.

        Can be a list of colors defined by name, RGB or RGBA.
        """
```

(same with other similar functions)

Reply:
I think it makes sense (it's been on the to do list). I can work on it later today!

Reply:
Done!